bin/git-theta

```diff
@@ -108,6 +124,18 @@ def track(args):
     git_utils.add_file(gitattributes_file, repo)
 
 
+def add(args, unparsed_args):
```

Comment:
What is the point of still having this `git theta add` command? Is it the only way to include user input like update type?

Does this mean to do a sparse update you need to do `git theta add` but `git add` will always assume a dense update?

Reply:
The intention was for this to be a convenience command for specifying update type. Right now we're giving this information to git-theta through environment variables but it seems more convenient to run `git theta add model.pt --update-type sparse` than `GIT_THETA_UPDATE_TYPE=sparse; git add model.pt`. The default behavior for when the environment variable isn't set is just dense.